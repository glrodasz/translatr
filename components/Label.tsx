import React from "react";

interface LabelProps {
  count: number;
  maxCount: number;
}

const Label: React.FC<LabelProps> = ({ count, maxCount }) => {
  return (
    <div className="h-6 text-right mt-2 text-sm text-gray-500 dark:text-gray-300">
      {count} / {maxCount}
    </div>
  );
};

export default Label;
