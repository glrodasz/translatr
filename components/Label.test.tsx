import { describe, it, expect } from "vitest";
import { render } from "@testing-library/react";
import Label from "./Label";

describe("[ components / Label ]", () => {
  describe("when Label is rendered", () => {
    it("should render", () => {
      // Arrange
      const props = {
        count: 0,
        maxCount: 10,
      };

      // Act
      const { asFragment } = render(<Label {...props} />);

      // Assert
      expect(asFragment()).toMatchSnapshot();
    });
  });
});
