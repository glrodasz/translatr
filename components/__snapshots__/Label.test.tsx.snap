// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`[ components / Label ] > when Label is rendered > should render 1`] = `
<DocumentFragment>
  <div
    class="h-6 text-right mt-2 text-sm text-gray-500 dark:text-gray-300"
  >
    0 / 10
  </div>
</DocumentFragment>
`;
